lib/mailbox.lua

```diff
@@ -39,7 +47,8 @@ function MailBox:receive()
 end
 
 function MailBox:send(message)
-	return self.queue:push(string.pack("s", message))
+	self.queue:push(string.pack("s", message))
```

Comment:
shouldn't we return `push` result no more?

Reply:
it was wrong to begin with; `push` never returned a value.

Reply:
unless you want to foresee potential future changes, like in that other point you came up with. I find it misleading, but it's really up to you here.

Reply:
No need.. the thing is `push()` used to return but now it raises an error.. all good 💪🏿